include/fmt/format-inl.h

```diff
@@ -935,8 +948,8 @@ template <typename Double> FMT_FUNC void fallback_format(Double v, int exp10) {
 
 template <typename Double,
           enable_if_t<(sizeof(Double) == sizeof(uint64_t)), int>>
-FMT_API bool grisu_format(Double value, buffer<char>& buf, int precision,
-                          unsigned options, int& exp) {
+FMT_API bool grisu_format(Double value, size_t sizeof_value, buffer<char>& buf,
```

Comment:
Since `sizeof_value` is only used to pass one bit of information (whether argument is `float` or `double`) I suggest folding it into `options`.